app/tetwild/TriangleInsertion.cpp

```diff
@@ -83,9 +83,10 @@ bool tetwild::TetWild::InputSurface::remove_duplicates() {
     return true;
 }
 
-void tetwild::TetWild::construct_background_mesh(const InputSurface &input_surface) {
-    const auto &vertices = input_surface.vertices;
-    const auto &faces = input_surface.faces;
+void tetwild::TetWild::construct_background_mesh(const InputSurface &input_surface)
```

Comment:
Why do you keep implementing new stuff instead of cleaning up the PR so we can merge it into the main branch?!